hits_cpp/hits.cpp

```diff
@@ -212,11 +212,34 @@ MemeRatingMap getRatings(Memes& memes, MemeShufflesMap& shuffles, bool verbose,
     return ratings;
 }
 
-void setHitsFactor(float factor, Memes& memes){
+float getRpmFactor(SequencesMap sequences) {
+    int count = 0;
+    float factor;
+    auto sequencesTypesIter = sequences.begin();
+    auto sequencesTypesEnd = sequences.end();
+
+    while (sequencesTypesIter != sequencesTypesEnd) {
+        auto sequencesIter = sequencesTypesIter->second->begin();
+        auto sequencesEnd = sequencesTypesIter->second->end();
+        while (sequencesIter != sequencesEnd) {
+            count++;
+            sequencesIter++;
+        }
+        sequencesTypesIter++;
+    }
+    cout << "size of all sequences: " << count << endl;
```

Comment:
You can keep the count on initialization (and as member).
This function should only do the factor division